client/keys/add.go

```diff
@@ -37,6 +37,8 @@ const (
 	flagKeyAlgo     = "algo"
 
 	// DefaultKeyPass contains the default key password for genesis transactions
+	// XXX What is this?!
+	// It's being used for the signing cli!
```

Comment:
It's no longer used. With the new keyring passwords are handled outside of the SDK